### DISCUSSION TITLE
Filmbox Negative/Print Multi-node Workflow - What colour space / log should be used in between for colour space aware tools?

### DISCUSSION THREAD

**Discussion by francisqureshi** [videovillage/community#187] [Q&A] [answered]
I quite like using Filmbox in the Neg / Print mode, with other correctors in the middle.. but I would love to know what's the best course of action for using the Resolve's HDR tool palette in between the Neg and Print - or say using a Tetra DCTL or pulling keys etc. 

I think I read a while back ACES**cc** was a good choice as it doesn't have the toe of ACES**cct** but would love to know what's advised by Video Village?

I guess my over arching question is what is the best practice? Maybe I am over complicating things and I should just be doing  major parallel corrections upstream before the Neg node still? Am I trying to work too closely to a Cineon style workflow?

This is a simplified Tree I've been using, Timeline space is usually Rec.709 Gamma 2.4 - SDR Commericial grading style 
  
![Screenshot from 2024-04-22 15-45-06](https://github.com/videovillage/community/assets/1946212/b98d91dc-7764-434a-9005-b2adc1642e68)

**Comment by gregcotten** [comment]
@andrewfinch might have a better answer than me but I think probably Cineon would be the closest match.

**Comment by Will35** [comment]
This is very similar to my node tree，do you know the answer now?

**Comment by andrewfinch** [comment, marked as answer]
Scene-referred / colorspace-aware grade operations should be done before the negative emulation. Keying will also work much better before the spatial processing of the negative has been applied.  After the negative node the image is just like a log negative scan, you could maybe describe it as being roughly Cineon but the main point of it is to do WYSIWYG creative operations or to use an Offset operation to emulate the effect of printer light adjustments that embrace the toe characteristics of the negative.

**Reply by andrewfinch** [reply]
(In theory setting Resolve's HDR tool palette to ACEScc and changing exposure would be identical to a traditional offset operation.)
